Rubberduck.Core/UI/ToDoItems/ToDoExplorerViewModel.cs

```diff
@@ -49,6 +49,12 @@ public sealed class ToDoExplorerViewModel : ViewModelBase, INavigateSelection, I
             _uiDispatcher = uiDispatcher;
             _state.StateChanged += HandleStateChanged;
 
+            RefreshCommand = new DelegateCommand(LogManager.GetCurrentClassLogger(),
+                _ =>
+                {
+                    _state.OnParseRequested(this);
+                },
+                _ => _state.IsDirty());
```

Comment:
This is the wrong can execute condition. The command should not be activated in most parser states. Moreover, it should always be available in `Ready`, `Error`, `ResolverError` and `Pending`.